types/lodash/index.d.ts

```diff
@@ -12269,7 +12269,7 @@ declare namespace _ {
          * @param value The value to inspect.
          * @return Returns true if value is empty, else false.
          */
-        isEmpty(value?: any): boolean;
+        isEmpty(value?: any): value is null | undefined;
```

Comment:
It wouldn't necessarily be caught:
```ts
import _ = require("lodash");

function intersection<T, U>(m1: Map<string, T>, m2: Map<string, U>): Map<string, [T, U]> {
	const result = new Map<string, [T, U]>();
	m1.forEach((v1, k) => {
		const v2 = m2.get(k);
		if (!_.isEmpty(v2))
			result.set(k, [v1, v2]); // This would no longer be a compile error.
	});
	return result;
}

const i = intersection(new Map([["k", 1]]), new Map([["k", {}]]));
console.log(i); // Expect "k" -> [1, {}], but it's empty???
```